packages/features/shell/Shell.tsx

```diff
@@ -339,8 +342,8 @@ function UserDropdown({ small }: { small?: boolean }) {
                   <span className="text-default truncate pb-1 font-normal leading-none">
                     {user.username
                       ? process.env.NEXT_PUBLIC_WEBSITE_URL === "https://cal.com"
-                        ? `cal.com/${user.username}`
-                        : `/${user.username}`
+                        ? `${orgBranding && orgBranding.slug}cal.com/${user.username}`
+                        : `${orgBranding && orgBranding.slug}/${user.username}`
```

Comment:
I was hoping to use `org.localhost:3000/username` but it was too long and it doesn't apply for staging, so left it with just `org/username` for reference. Prod will be set as expected `org.cal.com/username`